projects/js-packages/components/components/jetpack-footer/style.scss

```diff
@@ -1,116 +1,99 @@
 .jp-dashboard-footer {
```

Comment:
Putting all links in a list and using flexboxes allows us to get rid of a bunch of rules.